### PR TITLE
Update policy-control-diagnostic-data-mobile.md

### PR DESCRIPTION
**Pull request by tiwarisakshi02**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 1]
```diff
@@ -589,6 +589,7 @@ For more information about diagnostic data, including how to control what diagno
 - **hide** - Hide chat.
 - **hideChannel** - Hide a channel from the teams and channel list.
 - **image** - Image.
+- **inAppNotification**- Triggered when notification is tapped while user is active in the app
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:592]:
```suggestion
- **inAppNotification**- Triggered when a notification is tapped while the user is active in the app.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 2]
```diff
@@ -785,6 +786,7 @@ For more information about diagnostic data, including how to control what diagno
 - **notBlockedDevice** - A user doesn't reach the threshold for background activity failures in 30 days.
 - **notNow** - **Not now** is selected in reminder.
 - **notNowUpdate** - UpdateDefer.
+- **notification/ notification_clicked** – Triggered when notification is tapped
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:789]:
```suggestion
- **notification/ notification_clicked** – Triggered when a notification is tapped.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 3]
```diff
@@ -1038,14 +1040,17 @@ For more information about diagnostic data, including how to control what diagno
 - **showCard** - Tap on card buttons. Cards are key platform constructs and measuring their usage and pattern is necessary to understand platform usage and keep a look out for potential issues on the client side.
 - **shownReadReceiptNotice** - The user shown feature notice with settings options.
 - **signIn** - **Sign in** is selected on welcome page, or the **Sign In** button is tapped.
+- **SignInWithOTP** - User selects the option to sign in as a guest with one time passcode (OTP).
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1043]:
```suggestion
- **SignInWithOTP** - User selected the option to sign in as a guest with one time passcode (OTP). 
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 4]
```diff
@@ -1038,14 +1040,17 @@ For more information about diagnostic data, including how to control what diagno
 - **showCard** - Tap on card buttons. Cards are key platform constructs and measuring their usage and pattern is necessary to understand platform usage and keep a look out for potential issues on the client side.
 - **shownReadReceiptNotice** - The user shown feature notice with settings options.
 - **signIn** - **Sign in** is selected on welcome page, or the **Sign In** button is tapped.
+- **SignInWithOTP** - User selects the option to sign in as a guest with one time passcode (OTP). 
 - **signUp** - **Create a free account** or **Sign up for free** is selected.
+- **SignUpFromSignIn**- User taps on **Create a new account** option from sign-in
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1045]:
```suggestion
- **SignUpFromSignIn**- User taps on **Create a new account** option from sign-in.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 5]
```diff
@@ -1038,14 +1040,17 @@ For more information about diagnostic data, including how to control what diagno
...
 - **simultaneousCallForward** - Triggered when:
   - Simultaneous call forward target is set.
   - Simultaneous call forwarding is enabled (Calls ring me is enabled & Also ring is set).
 - **skipVerificationForLink** - Thw user chose to skip verification.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1049]:
```suggestion
- **skipVerificationForLink** - The user chose to skip verification.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 6]
```diff
@@ -1038,14 +1040,17 @@ For more information about diagnostic data, including how to control what diagno
...
 - **smartReply** - Smart reply toggle button is clicked.
 - **SMSSendMessage** - The user sends a SMS message.
 - **sortChanged** - Triggers when user changes sort order while viewing a tasklist.
+- **SSOAccountListItem**: Triggered when user taps on an SSO account to sign-in
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1053]:
```suggestion
- **SSOAccountListItem**: Triggered when the user taps on an SSO account to sign in.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 7]
```diff
@@ -1073,6 +1078,7 @@ For more information about diagnostic data, including how to control what diagno
 - **stuckOnConnectingRetrySelected** - **Retry** is selected on the drawer.
 - **stuckOnConnectingShownDismissed** - A user dismissed the drawer.
 - **suggested_place_selected** - A user shares a static location by selecting a suggested place.
+- **Switching**- Tenant or account is switched from the app. This is required to measure account/ tenant switch issues proactively and provide smoot account/ tenant switch experience.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1081]:
```suggestion
- **Switching**- Tenant or account is switched from the app. This is required to measure account/tenant switch issues proactively and provides a smooth account/tenant switch experience.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 8]
```diff
@@ -1232,18 +1239,28 @@ action.
 - **meetingFiles** - Triggered when meeting files screen is opened.
 - **meetNowActionSheet** - Triggered when user creates a Meet Now meeting.
 - **navPersonalFiles** - Triggered when navigation to files screen is performed.
+- **signInSSOPage**: Triggered when user views single sign on page while sign in
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1242]:
```suggestion
- **signInSSOPage**: Triggered when the user views a single sign-on page while signed in.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 9]
```diff
@@ -1232,18 +1239,28 @@ action.
...
+-- **signInError**: Triggered when user hits any error while sign in. This is needed to proactively identify and fix issues that users face during sign in.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1243]:
```suggestion
-- **signInError**: Triggered when the user hits any error while signed in. This is needed to proactively identify and fix issues that users face during sign-in. 
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 10]
```diff
@@ -1232,18 +1239,28 @@ action.
 - **meetingFiles** - Triggered when meeting files screen is opened.
 - **meetNowActionSheet** - Triggered when user creates a Meet Now meeting.
 - **navPersonalFiles** - Triggered when navigation to files screen is performed.
+- **signInSSOPage**: Triggered when user views single sign on page while sign in
+-- **signInError**: Triggered when user hits any error while sign in. This is needed to proactively identify and fix issues that users face during sign in. 
+-- **TfLSignInSuccessful** : Triggered when user successfully signs in to personal Microsoft account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1244]:
```suggestion
-- **TfLSignInSuccessful**: Triggered when the user successfully signs in to a personal Microsoft account. This is needed to understand sign-in and sign-up reliability and proactively identify and fix issues.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 11]
```diff
@@ -1232,18 +1239,28 @@ action.
...
+-- **TfWFreemiumSignInSuccessful** : Triggered user successfully signs in to freemium account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1245]:
```suggestion
-- **TfWFreemiumSignInSuccessful**: Triggered when the user successfully signs in to a freemium account. This is needed to understand sign-in and sign-up reliability and proactively identify and fix issues.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 12]
```diff
@@ -1232,18 +1239,28 @@ action.
...
+-- **TfWSignInSuccessful** : Triggered when user successfully signs in to work or school account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1246]:
```suggestion
-- **TfWSignInSuccessful**: Triggered when the user successfully signs in to a work or school account. This is needed to understand sign-in and sign-up reliability and proactively identify and fix issues.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 13]
```diff
@@ -1232,18 +1239,28 @@ action.
...
 
 ### Scenario
 
 > [!NOTE]
 > For information on the properties of PanelAction events, see [Properties sent with scenario events](#properties-sent-with-scenario-events).
-
+> 
+-- **acquire_resource_token_interactive**- Required service call which is triggered when authentication token is acquired by interactive sign in.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1253]:
```suggestion
-- **acquire_resource_token_interactive**- Required service call which is triggered when an authentication token is acquired by interactive sign-in. 
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 14]
```diff
@@ -1232,18 +1239,28 @@ action.
...
+-- **TfWSignInSuccessful** : Triggered when user successfully signs in to work or school account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.
 
 ### Scenario
 
 > [!NOTE]
 > For information on the properties of PanelAction events, see [Properties sent with scenario events](#properties-sent-with-scenario-events).
-
+> 
+-- **acquire_resource_token_interactive**- Required service call which is triggered when authentication token is acquired by interactive sign in. 
+-- **acquire_resource_token_silent**- Required service call which is triggered when authentication token is acquired by silent sign in.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1254]:
```suggestion
-- **acquire_resource_token_silent**- Required service call which is triggered when an authentication token is acquired by silent sign-in.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 15]
```diff
@@ -1232,18 +1239,28 @@ action.
...
+-- **app_crash2** – Triggered when app is crashed unexpectedly. Provides information on how frequently Teams app is crashing.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1255]:
```suggestion
-- **app_crash2** – Triggered when the app is crashed unexpectedly. Provides information on how frequently the Teams app is crashing. 
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 16]
```diff
@@ -1232,18 +1239,28 @@ action.
...
+> 
+-- **acquire_resource_token_interactive**- Required service call which is triggered when authentication token is acquired by interactive sign in. 
+-- **acquire_resource_token_silent**- Required service call which is triggered when authentication token is acquired by silent sign in.
+-- **app_crash2** – Triggered when app is crashed unexpectedly. Provides information on how frequently Teams app is crashing. 
 - **app_incremental_sync_launch** Confirms that the pill count gets updated successfully for cold launch.
 - **app_incremental_sync_resume** Confirms that the pill count gets updated successfully for warm/hot launch.
 - **app_start_cold** To monitor cold app launch (Android only).
 - **app_start_hot** To monitor hot app launch (Android only).
 - **app_start_warm** To monitor warm app launch (Android only).
+- **auth_adal_tokens**- Required service call to do silent authentication. Triggered when user starts the app, or token is refreshed on expiry.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1261]:
```suggestion
- **auth_adal_tokens**- Required service call to do silent authentication. Triggered when a user starts the app, or the token is refreshed on expiry.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 17]
```diff
@@ -1232,18 +1239,28 @@ action.
...
 - **chat_add_giphy** - Confirms that the Giphy GIF rendering action succeeded or failed.
+- **chat_send_message_sfc**- Triggered when chat message is sent in Sfc interop chat
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1263]:
```suggestion
- **chat_send_message_sfc**- Triggered when a chat message is sent in SfC interop chat.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 18]
```diff
@@ -1232,18 +1239,28 @@ action.
...
 - **cortanaError** To monitor Cortana error happens.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1264]:
```suggestion
- **cortanaError** - To monitor Cortana error happens.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 19]
```diff
@@ -1232,18 +1239,28 @@ action.
...
 - **cortanaView** - To monitor Cortana canvas appear.
 - **cortanaRestart** To monitor Cortana restart.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1266]:
```suggestion
- **cortanaRestart** - To monitor Cortana restart.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 20]
```diff
@@ -1263,6 +1280,7 @@ action.
 - **cortana_skill_action_delay** - Confirms the start of delay action.
 - **cortana_watchdog** - To monitor Cortana watchdog recovery process.
 - **create_default_plan_and_nav_to_view** - Confirms successful creation of a default shared task list and how long it took for a user to land on the resulting view after action.
+- **create_new_chat_thread_sfc**- Triggered when new chat thread is created for sfc interop chat
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1283]:
```suggestion
- **create_new_chat_thread_sfc**- Triggered when a new chat thread is created for an SfC interop chat.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 21]
```diff
@@ -1272,7 +1290,12 @@ action.
 - **delete_personal_task** - Confirms the successful deletion of a personal task item.
 - **delete_planner_plan** - Confirms the successful deletion of a shared task list.
 - **delete_planner_task** - Confirms the successful deletion of a shared task item.
+- **json_parse_failure**- Provides information on the frequently of JSON parsing issues.
+- **getProfilePicture**- Necessary service call to get user profile picture. 
+- **get_resource_token_async**: Required service call to acquires tokens for Azure Active Directory resources asynchronously
+- **get_resource_token_sync**: Required service call to acquires tokens for Azure Active Directory resources synchronously
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1296]:
```suggestion
- **get_resource_token_async**: Required service call to acquire tokens for Azure Active Directory resources asynchronously.
- **get_resource_token_sync**: Required service call to acquires tokens for Azure Active Directory resources synchronously.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 22]
```diff
@@ -1272,7 +1290,12 @@ action.
...
 - **get_sender_sub_scenario** - get sender sub scenario in activity.
+- **interactiveAuthNopa2** – Triggered when no password user is interrupted to do interactive authentication
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1298]:
```suggestion
- **get_sender_sub_scenario** - Get sender sub scenario in activity.
- **interactiveAuthNopa2** – Triggered when no password user is interrupted to do interactive authentication.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 23]
```diff
@@ -1287,6 +1310,7 @@ action.
 - **rename_personal_plan** - Confirms the successful renaming of a personal task list.
 - **rename_planner_plan** - Confirms the successful renaming of a shared task list.
 - **save_image** Confirms that the image save action succeeded or failed.
+- **saveMeProfile**- Required service call that gets triggered when user saves the profile
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1313]:
```suggestion
- **save_image** - Confirms that the image save action succeeded or failed.
- **saveMeProfile**- Required service call that gets triggered when the user saves the profile.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 24]
```diff
@@ -1287,6 +1310,7 @@ action.
...
 - **share_image** Confirms that the image share action succeeded or failed.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1314]:
```suggestion
- **share_image** - Confirms that the image share action succeeded or failed.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 25]
```diff
@@ -1311,6 +1335,7 @@ action.
 - **meeting_details** Confirms that the meeting details sync has succeeded or failed.
 - **show_meeting_participants** Confirms that showing the meeting participant list has succeeded or failed.
 - **search** Confirms that the whole search session has succeeded or failed.
+- **time_based_retention_shared_channel** – Captures performance data for pruning the database
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1338]:
```suggestion
- **meeting_details** - Confirms that the meeting details sync has succeeded or failed.
- **show_meeting_participants** - Confirms that showing the meeting participant list has succeeded or failed.
- **search** - Confirms that the whole search session has succeeded or failed.
- **time_based_retention_shared_channel** – Captures performance data for pruning the database.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tiwarisakshi02, kindly commit my suggested changes. Many thanks.

**Pull request merged**